pkg/kubelet/eviction/helpers.go

```diff
@@ -18,20 +18,19 @@ package eviction
 
 import (
 	"fmt"
-	"sort"
-	"strconv"
-	"strings"
-	"time"
-
 	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	"k8s.io/klog"
+	"k8s.io/kubernetes/pkg/apis/scheduling"
 	"k8s.io/kubernetes/pkg/features"
 	statsapi "k8s.io/kubernetes/pkg/kubelet/apis/stats/v1alpha1"
 	evictionapi "k8s.io/kubernetes/pkg/kubelet/eviction/api"
 	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
-	schedulerutils "k8s.io/kubernetes/pkg/scheduler/util"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
```

Comment:
can you please keep the order as it was before? those imports should be at the top.

Reply:
+1
I'm surprised lint didn't complain about this.